crate_universe/private/vendor_utils.bzl

```diff
@@ -14,10 +14,13 @@ _BUILDIFIER_SHA256S = {
 }
 
 def crates_vendor_deps():
```

Comment:
CI failure:

```console
##### :bazel: buildifier: found 1 lint issue in your WORKSPACE, BUILD and *.bzl files
crate_universe/private/vendor_utils.bzl:16:1: function-docstring: The function "crates_vendor_deps" has no docstring.
A docstring is a string literal (not a comment) which should be the first statement of a function body (it may follow comment lines).
```

Presumably this is now needed because the function no longer returns void.